aboutViewer.js

```diff
@@ -0,0 +1,46 @@
+var repos = {};
+repos.all = [];
+repos.branches = [];
+
+repos.RequestAll = function (callback){
+
+  $.ajax({
+    type:'GET',
+    url: 'https://api.github.com/users/MiaRomero/repos',
+    headers: { Authorization: 'token ' + githubToken }
+
+  }).done(function(data){
+    console.log(data);
+    repos.all = data;
+  }).done(callback);
+};
```

Comment:
You may want to consider adding some .success() and .fail() methods here. Currently if this fails, nothing happens, and your user will be left scratching their head.